$NetBSD: patch-ab,v 1.2 2009/04/07 13:19:20 taca Exp $

Additional Japanese translation for tinyMCE.  It will included
in TYPOlight 2.7.0 release.

--- plugins/tinyMCE/plugins/typolinks/langs/ja.js.orig	2009-03-30 14:02:00.000000000 +0900
+++ plugins/tinyMCE/plugins/typolinks/langs/ja.js
@@ -0,0 +1,4 @@
+tinyMCE.addI18n('ja.typolinks',{
+link_desc:'リンクの挿入・編集',
+image_desc:'リンクの挿入・編集のライトボックス画像'
+});
